org.eclipse.scanning.api/src/org/eclipse/scanning/api/device/DeviceResponse.java

```diff
@@ -116,35 +120,35 @@ private static void processScannables(DeviceRequest request, IScannableDeviceSer
 					 * for this call to return before position events are sent out.
 					 * FUDGE warning
 					 */
-					Thread.sleep(10); 
+					Thread.sleep(10);
 					 /* End warning */
 				}
 			} else if (action==DeviceAction.ACTIVATE) {
 				device.setActivated((Boolean)request.getDeviceValue());
 			}
-			
-			
+
+
 			if (action!=null && action.isTerminate() && device instanceof ITerminatable) {
 				ITerminatable tdevice = (ITerminatable)device;
 				tdevice.terminate(action.to());
 			}
-			
+
 			request.setDeviceValue(device.getPosition());
 
 			DeviceInformation<?> info = new DeviceInformation<Object>(device.getName());
 			merge(info, device);
 			request.addDeviceInformation(info);
-			
+
 		} else {
 			final Collection<String> names = cservice.getScannableNames();
 			for (String name : names) {

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reduce the total number of break and continue statements in this loop to use at most one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS135)